tests/sles4sap/trento/cluster_configure.pm

```diff
@@ -26,7 +26,7 @@ sub run {
 sub post_fail_hook {
     my ($self) = shift;
     $self->select_serial_terminal;
-    qesap_upload_logs('', 1);
+    qesap_upload_logs();
```

Comment:
I thought `upload_logs` has by default failok enabled, so I thought you can have it by default enabled and don't need to use arguments. But upload of logs should not fail. :innocent: